research-update.md

```diff
@@ -0,0 +1,29 @@
+ 
+### This document is NOT:
+* Finished or complete
+* A list of everything we plan to fix or address
+* Quantitative, statistically significant, or generalizable
+
+### This document IS:
+* Initial research that is broader than what we plan to work on
+* Imperfect and incomplete: please let us know what we’re missing!
+* Qualitative: this research is used to find patterns to further examine and test
+ 
+**We are sharing our initial research because we know it is imperfect and would greatly appreciate your feedback. Please [open an issue](https://github.com/18F/foia-recommendations/issues/new) in GitHub or email us at [National.FOIAPortal@usdoj.gov](mailto:National.FOIAPortal@usdoj.gov), and keep our [contributing policy](https://github.com/18F/foia-recommendations/blob/master/CONTRIBUTING.md) in mind.**
+ 
+### Interviews
+We started with a first round of interviews with users and stakeholders, including agencies, requesters, requester advocates, and vendors. (A HUGE thank you to everyone who took time to speak with us so far!) To learn more, read [our research plan](https://github.com/18F/foia-recommendations/blob/master/research-plan.md) or [18F’s approach to stakeholder and user interviews](https://methods.18f.gov/discover/stakeholder-and-user-interviews/).
+ 
+Next, we synthesized our interviews to identify common patterns and pain points, including for [requesters](https://github.com/18F/foia-recommendations/blob/research-synthesis-requesters.md) and [agencies](https://github.com/18F/foia-recommendations/blob/research-synthesis-agencies.md). Importantly, these syntheses are early possible findings that we plan to test and research further, not a final report.
+ 
+### Hypotheses and user stories
+Currently, we are brainstorming hypotheses and user stories based on our research so far. Next, we’ll prioritize them based on what we understand the least and would pose the greatest risk to the project if we got them wrong. This is based on [lean methodologies](https://lean-product-design.18f.gov/). 
+ 
+### Prototypes
+We’ll start building prototypes in this GitHub repo to test those hypotheses with users and stakeholders soon. A good software prototype is like a movie set: they’re quicker and easier to build than the real thing, but just believable enough to show off (for testing). Learn more about [18F’s approach to prototyping](https://methods.18f.gov/make/prototyping/) and [usability testing](https://methods.18f.gov/validate/usability-testing/).
+ 
+### Feedback
+Your feedback is valuable in helping us adjust and improve our hypotheses. We are sharing our initial research because we know it is imperfect and would greatly appreciate your feedback. Please [open an issue](https://github.com/18F/foia-recommendations/issues/new) in GitHub or email us at [National.FOIAPortal@usdoj.gov](mailto:National.FOIAPortal@usdoj.gov), and keep our [contributing policy](https://github.com/18F/foia-recommendations/blob/master/CONTRIBUTING.md) in mind.
```

Comment:
Maybe a little redundant, since this is mentioned in bold above, but I like having the bold piece, and I like that this paragraph makes feedback an important part of the process, so idk.